libraries/RC_Channel/RC_Channel.h

```diff
@@ -424,8 +424,8 @@ class RC_Channels {
         IGNORE_FAILSAFE       = (1 << 2), // ignore RC failsafe bits
         FPORT_PAD             = (1 << 3), // pad fport telem output
         LOG_DATA              = (1 << 4), // log rc input bytes
-        ARMING_CHECK_THROTTLE = (1 << 5), // run an arming check for neutral throttle
```

Comment:
Umm.  Can we really just invert the meaning of this bit?

This is all persistent data.  So if someone has set the check-throttle bit then this will now skip that check...

@WickedShell did this stuff.

Reply:
Oh, didn't think about whether anyone was actually enabling this check.
Do you see any problems with just changing the default to (1<<5)?